src/app/filecoin-plus/allocators/components/NoAllocatorsComponents.tsx

```diff
@@ -0,0 +1,19 @@
+import { FIL_PLUS_URLS } from '@/constants/siteMetadata'
+
+import { Button } from '@/components/Button'
+
+export function NoAllocatorsComponent() {
```

Comment:
We have a `NoResultsMessage` and `ErrorMessage` component - They probably don't overlap enough but something to look into.

Perhaps we can at least use `Heading`?